docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+services:
+    postgres:
+        build: ./docker/postgres/
+        environment:
+            POSTGRES_USER: kukkuu
+            POSTGRES_PASSWORD: kukkuu
+            POSTGRES_DB: kukkuu
+        ports:
+            - "5432:5432"
+        volumes:
+            - kukkuu-postgres-data-volume:/var/lib/postgresql/data
```

Comment:
if you name the volume just `postgres-data-volume` that will be enough, as docker engine will add the root folder's name as prefix anyway - `kukkuu_postgres-data-volume`. Guilty of excessive prefixes myself 😅

Reply:
Makes sense, changing.